libraries/stdlib/api/js/kotlin.kt

```diff
@@ -899,6 +909,8 @@ public final class ByteArray {
 }
 
 public final class Char : kotlin.Comparable<kotlin.Char> {
+/*∆*/     public constructor Char(code: kotlin.UShort)
```

Comment:
Please annotate the new constructor as experimental, if possible

Reply:
+1